gradle/release.gradle

```diff
@@ -43,14 +48,9 @@ notes {
     ]
 }
 
-def dryRun = project.hasProperty('dryRun')
-if (dryRun) {
-    //TODO SF lets not use excludedTaskNames to model this
-    logger.lifecycle "Automatically scheduling 'rollbackRelease' task"
-    //mustRun instead of dependsOn to avoid running 'release' task every time use 'dryRun' property
-    rollbackRelease.mustRunAfter release
-    gradle.startParameter.taskNames += "rollbackRelease"
-}
+//dryRun project property is interpreted by the release-workflow plugin to enable rollbacks
+//releaseTest does not publish or push but it leaves the commits so that they can be inspected manually
+def releaseTest = project.hasProperty('releaseTest') || project.hasProperty('dryRun')
```

Comment:
Maybe rename to `releaseWorkflowTest` otherwise it maybe understood as run test against release (for users that don't know much about gradle but that have shell completion)